community/rust/APKBUILD

```diff
@@ -161,6 +164,17 @@ lldb() {
 	_mv "$pkgdir"/$_sharedir/etc/lldb_*.py $_sharedir/etc/
 }
 
+src() {
+	pkgdesc="Rust source code for ycmd and racerd code completion support"
+	depends="$pkgname"
+	license="$license OFL-1.1 GPL-3.0-or-later GPL-3.0-with-GCC-exception CC-BY-SA-3.0 LGPL-3.0"
```

Comment:
Where is this list of licenses from?

Reply:
They are in the src folder.  You can just grep for them.  We could put them in a doc subpackage.

Reply:
Then please add them to top-level `license=`. I guess that licenses should be the same for both source code and binary built from the source code.

Reply:
> They are in the src folder. You can just grep for them.

Could you please describe how exactly did you extracted this licenses list? I don’t see them.

Also:
[rust README.md](https://github.com/rust-lang/rust#license):
> Rust is primarily distributed under the terms of both the MIT license and the Apache License (Version 2.0), with portions covered by various BSD-like licenses.

[Fedora rust.spec](https://src.fedoraproject.org/rpms/rust/blob/f27/f/rust.spec#_53-54):
> License:        (ASL 2.0 or MIT) and (BSD and ISC and MIT)
>  ^ written as: (rust itself) and (bundled libraries)

There are many bundled build-only dependencies, so maybe this explains the extra licenses you found?

Reply:
Meanwhile I merged addition of -src subpackage with your original suggestion of licenses.